Scenes/MainScenes/Game.cs

```diff
@@ -164,10 +164,27 @@ public override void _Ready()
     fireControl = gameUI.GetNode<Label>("ControlIndicators/ControlsBox/FireButton");
 
     PackedScene packedLoginScene = (PackedScene)ResourceLoader.Load("res://Scenes/LoginScreen.tscn");
+
     loginScreen = (LoginScreen)packedLoginScene.Instance();
-    loginScreen.Hide();
     this.AddChild(loginScreen);
-    loginScreen.Show();
+
+ var clientConfig = new ConfigFile();
+
+	Godot.Error err = clientConfig.Load("res://Resources/client.cfg");	  
+	var  activateAuthDev = (Boolean)clientConfig.GetValue("auth", "activate_auth_dev");
+
+	if (OS.IsDebugBuild() && activateAuthDev == false)
+	{ 
+		this._on_go_to_game(true);
+		return;
+	}
```

Comment:
Indentation

Reply:
Also, there is already a `LoadConfig()` method -- shouldn't this config loading be part of that method?